teams/wg-async-foundations.toml

```diff
@@ -0,0 +1,19 @@
+name = "wg-async-foundations"
+wg = true
```

Comment:
oops, the whole point of this was to sync with a github team, I should add that. ( I want to use give that team ownership of https://github.com/rust-lang/wg-async-foundations )